layouts/_default/single.html

```diff
@@ -4,6 +4,11 @@
 	<article class="post">
 		<header class="post__header">
 			<h1 class="post__title">{{ .Title }}</h1>
+			{{- if .Params.lead }}
```

Comment:
Question: what about `{{ with }}` instead of `{{ if }}`? Or I missed something?

Reply:
You're right